backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/CodeGenerator.kt

```diff
@@ -935,12 +891,33 @@ internal class FunctionGenerationContext(val function: LLVMValueRef,
                 )
             }
         }
+
+        // If object is imported - access it via getter function.
+        if (isExternal(irClass)) {
+            val valueGetterName = irClass.objectInstanceGetterSymbolName
+            val valueGetterFunction = LLVMGetNamedFunction(context.llvmModule, valueGetterName) ?:
+                LLVMAddFunction(context.llvmModule, valueGetterName,
+                        LLVMFunctionType(kObjHeaderPtr, cValuesOf(kObjHeaderPtrPtr), 1, 0))
```

Comment:
`functionType` function is generally a "kotlinized" shortcut for `LLVMFunctionType`.